nbgrader/server_extensions/formgrader/templates/formgrade.tpl

```diff
@@ -13,26 +14,8 @@
     </style>
 {% endfor %}
 
-<!-- MathJax -->
-<script type="text/javascript">
-window.MathJax = {
-    tex2jax: {
-        inlineMath: [ ['$','$'], ["\\(","\\)"] ],
-        displayMath: [ ['$$','$$'], ["\\[","\\]"] ],
-        processEscapes: true,
-        processEnvironments: true
-    },
-    // Center justify equations in code and markdown cells. Elsewhere
-    // we use CSS to left justify single line equations in code cells.
-    displayAlign: 'center',
-    "HTML-CSS": {
-        styles: {'.MathJax_Display': {"margin": 0}},
-        linebreaks: { automatic: true }
-    }
-};
-</script>
-
-<script type="text/javascript" src="{{ resources.base_url }}/{{ resources.mathjax_url }}?config=TeX-AMS-MML_HTMLorMML-full"></script>
+<!-- Loading mathjax macro -->
+{{ mathjax() }}
```

Comment:
This removes the ability to specify a custom mathjax url, so maybe change this to:

```
{{ mathjax('{{ resources.base_url }}/{{ resources.mathjax_url }}?config=TeX-AMS-MML_HTMLorMML-full') }}
```

I am not 100% sure that's the right syntax but I think something along those lines should work looking at the macro arguments here: https://github.com/jupyter/nbconvert/blob/master/nbconvert/templates/html/mathjax.tpl

Reply:
You're right. I've changed it. This destroys my "make formgrader and feedback use the exact same mathjax urls" plans, but after digging around a bit, I see that this idea never made much sense :)